aixplain/factories/agent_factory/__init__.py

```diff
@@ -50,8 +50,35 @@ def create(
         api_key: Text = config.TEAM_API_KEY,
         supplier: Union[Dict, Text, Supplier, int] = "aiXplain",
         version: Optional[Text] = None,
+        use_supervisor: bool = True,
```

Comment:
Should we rename here to `use_orchestrator`? Since the community will also have the orchestrator in its name.

Reply:
I removed this flag for now, since this option will always be in the architecture for now.